tests/data/collimator_fake.asdf

```diff
@@ -0,0 +1,40 @@
+#ASDF 1.0.0
```

Comment:
This file should stay in jwst/datamodels/tests. The test that uses it is commented out so it's safe to remove it.